addons/l10n_ie/data/template/account.tax-ie.csv

```diff
@@ -1,63 +1,197 @@
+,"","","","","","","","","tax","invoice","","l10n_ie_account_2131","+T2"
+,"","","","","","","","","base","refund","","",""
+,"","","","","","","","","tax","refund","","l10n_ie_account_2131","-T2"
+"ie_tax_purchase_good_0","0% G","0% Goods","0%","purchase","percent","0","ie_tax_group_vat_0","True","base","invoice","","",""
+,"","","","","","","","","tax","invoice","","","+T2"
+,"","","","","","","","","base","refund","","",""
+,"","","","","","","","","tax","refund","","","-T2"
+"ie_tax_purchase_service_0","0% S","0% Services","0%","purchase","percent","0","ie_tax_group_vat_0","True","base","invoice","","",""
+,"","","","","","","","","tax","invoice","","","+T2"
+,"","","","","","","","","base","refund","","",""
+,"","","","","","","","","tax","refund","","","-T2"
+"ie_tax_purchase_good_exempt","0% G EXEMPT","0% Goods Exempt","0%","purchase","percent","0","ie_tax_group_vat_0","True","base","invoice","","",""
+,"","","","","","","","","tax","invoice","","","+T2"
+,"","","","","","","","","base","refund","","",""
+,"","","","","","","","","tax","refund","","","-T2"
+"ie_tax_purchase_service_exempt","0% S EXEMPT","0% Services Exempt","0%","purchase","percent","0","ie_tax_group_vat_0","True","base","invoice","","",""
+,"","","","","","","","","tax","invoice","","","+T2"
+,"","","","","","","","","base","refund","","",""
+,"","","","","","","","","tax","refund","","","-T2"
+"ie_tax_purchase_eu_good_0","0% EU G","0% EU Goods","0%","purchase","percent","0","ie_tax_group_vat_0","True","base","invoice","","","+E2"
+,"","","","","","","","","tax","invoice","","","+T1"
+,"","","","","","","","","tax","invoice","","","+T2"
+,"","","","","","","","","base","refund","","","-E2"
+,"","","","","","","","","tax","refund","","","-T1"
+,"","","","","","","","","tax","refund","","","-T2"
+"ie_tax_purchase_eu_service_0","0% EU S","0% EU Services","0%","purchase","percent","0","ie_tax_group_vat_0","True","base","invoice","","","+ES2"
+,"","","","","","","","","tax","invoice","","","+T2"
+,"","","","","","","","","base","refund","","","-ES2"
+,"","","","","","","","","tax","refund","","","-T2"
+"ie_tax_purchase_ex_good_0","0% EX G","0% Import Goods","0%","purchase","percent","0","ie_tax_group_vat_0","True","base","invoice","","",""
+,"","","","","","","","","tax","invoice","","","+T2"
+,"","","","","","","","","base","refund","","",""
+,"","","","","","","","","tax","refund","","","-T2"
+"ie_tax_purchase_ex_service_0","0% EX S","0% Import Services","0%","purchase","percent","0","ie_tax_group_vat_0","True","base","invoice","","",""
+,"","","","","","","","","tax","invoice","","","+T2"
+,"","","","","","","","","base","refund","","",""
+,"","","","","","","","","tax","refund","","","-T2"
+"ie_tax_purchase_ex_good_pa_0","0% EX G PA","0% Import Goods - Postponed Accounting","0%","purchase","percent","0","ie_tax_group_vat_0","True","base","invoice","","","+PA1"
```

Comment:
goods import = postponed accounting because you get the customs invoice afterwards?

Reply:
https://www.revenue.ie/en/customs/businesses/brexit/info-businesses/vat-gb-after-transition/postponed-accounting.aspx

Reply:
https://www.revenue.ie/en/customs/businesses/brexit/info-businesses/vat-gb-after-transition/postponed-accounting.aspx